scripts/cy-wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
+export CY_API_URL="${CY_API_URL:-https://http-api.cycloid.io}"
+export CY_BINARIES_PATH="${CY_BINARIES_PATH:-$HOME/.cycloid}"
```

Comment:
would not be meaningful to download the binaries in the `XDG_CONFIG_HOME/cycloid-cli` or `$HOME/.config/cycloid-cli` where we already keep the config ?

Reply:
I would prefer `/usr/local/bin` which is often the de facto location to install custom binary tools (eg. `aws`, `scw`, etc.)

Reply:
Just to sum up, this wrapper will be used in our `cycloid-toolkit` docker image.
We don't recommend or expect users to use this on their laptops (we have another plan for them).

So our usage context should be a docker image used in a pipeline.

**pipeline** (wrapper)

The main goal is to be able to store the last X (3) version of our `cy` binary in cache (in the docker image).
Then the wrapper will use them. If not present it will download them.

**regular users**

My current idea for users is to migrate the download part in the actual CLI. The same way as fly for concourse.

The user will have only one `cy` binary. They don't need a wrapper to store and download X version of this binary

The user will run something like `cy sync` which will actually download the appropriated version of the binary vs the API you are using. And put it under your `cy` path
For now we just have a warning message and the user have to do it manually.

Reply:
Path have been changed to `/usr/local/bin`